docs/source/modules.rst

```diff
@@ -0,0 +1,7 @@
+pptx_blueprint
```

Comment:
Was this autogenerated? I don't think we need it for anything. And it makes sphinx moan:
~~~
/home/tim/dev/pptx-blueprint/docs/source/modules.rst: WARNING: document isn't included in any toctree
~~~